src/LibraryManager/Manifest.cs

```diff
@@ -97,6 +98,7 @@ internal static Manifest FromJson(string json, IDependencies dependencies)
         {
             try
             {
+                LibraryIdToNameAndVersionConverter.Instance.Initialize(dependencies);
```

Comment:
This doesn't seem like the right place to init a shared static instance of LibraryIdToNameAndVersionConverter. It would be fine if we had per-manifest instances of LibraryIdToNameAndVersionConverter, but we don't. Since all manifests share it, I don't believe each should re-initialize it. If we want to keep init here, it should be something like "EnsureInitialized" with an assumption that the set of providers is always the same for all manifests, and init doesn't get re-done every time. If the set of providers may be different for different manifests, then we should have per-manifest LibraryIdToNameAndVersionConverter instances.

Reply:
The providers will remain the same for all providers for now. 
So changing this to be `EnsureInitialized()` for now. 
I am going to open a new issue to change the converter to be per manifest instead (Its going to be a much bigger change).